docs/.vitepress/theme/custom.css

```
@@ -0,0 +1,38 @@
+:root {
+    --vp-home-hero-name-color: transparent;
+    --vp-home-hero-name-background: -webkit-linear-gradient(
```

Stylelint (17.4.0)

[error] Unexpected vendor-prefixed value "-webkit-linear-gradient" (value-no-vendor-prefix)

(value-no-vendor-prefix)